pallets/pallet-web3-names/Cargo.toml

```diff
@@ -0,0 +1,59 @@
+[package]
+authors = ["KILT <info@kilt.io>"]
+description = "Unique Web3 nicknames for KILT DIDs."
+edition = "2021"
+name = "pallet-web3-names"
+repository = "https://github.com/KILTprotocol/mashnet-node"
+version = "1.3.1"
```

Comment:
update to 1.4.0

Reply:
https://github.com/KILTprotocol/mashnet-node/pull/314/commits/0ea2b8624c0fcdf765b707a0a6462acbdf3d0c38